repository-data/application/src/main/resources/hcm-content/content/documents/administration/intranet/task.yaml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
Is this a `Task` resource bundle document?

Reply:
yes, i've just added the existing Task resource bundles so they can be used when devleoping/testing. They won't be bootstrapped into live as I've already had them manually imported there